test/parallel/test-whatwg-url-origin.js

```diff
@@ -42,7 +45,7 @@ function runURLTests(urltests) {
     var expected = urltests[i]
     if (typeof expected === "string" || !("origin" in expected)) continue
     test(function() {
-      var url = bURL(expected.input, expected.base)
+      var url = bURL(expected.input || expected.url, expected.base)
```

Comment:
This part is not in WPT, and the test data format is different actually. I think we can simply move it to the bottom and leave the WPT part alone for the ease of syncing/upstreaming.

```js
// Tests below are not from WPT.
const tests = require(path.join(common.fixturesDir, 'url-tests-additional.js'));
for (const test of tests) {
  if (typeof test === 'object' && 'origin' in test) {
    const url = new URL(test.url);
    assert.strictEqual(url.origin, test.origin);
  }
}
```